runtime/src/accounts_index.rs

```diff
@@ -450,12 +450,6 @@ pub struct RootsTracker {
     /// Updated every time we add a new root or clean/shrink an append vec into irrelevancy.
     /// Range is approximately the last N slots where N is # slots per epoch.
     pub(crate) alive_roots: RollingBitField,
-    /// Set of roots that are roots now or were roots at one point in time.
-    /// Range is approximately the last N slots where N is # slots per epoch.
-    /// A root could remain here if all entries in the append vec at that root are cleaned/shrunk and there are no
-    /// more entries for that slot. 'alive_roots' will no longer contain such roots.
-    /// This is a superset of 'alive_roots'
-    pub(crate) historical_roots: RollingBitField,
```

Comment:
Removing this is the main change. All other fns and tests should be about using this member.